qiita_core/support_files/server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Will the server key be automatically regenerated? Otherwise, distributing a
private key is usually not a good idea
On Dec 22, 2015 09:13, "Jose Navas" <notifications@github.com> wrote:

> In qiita_core/support_files/server.key
> <https://github.com/biocore/qiita/pull/1584#discussion_r48275147>:
>
> > @@ -0,0 +1,27 @@
> > +-----BEGIN RSA PRIVATE KEY-----
>
> Nope, the server needs the private key to run.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/1584/files#r48275147>.
>